src/util/rocksdb/properties.rs

```diff
@@ -252,6 +252,7 @@ impl IndexHandles {
     }
 
     fn get_approximate_distance_in_range(&self, start: &[u8], end: &[u8]) -> u64 {
+        assert!(end >= start);
```

Comment:
Does equal condition exist?

Reply:
Doesn't exist for now. But from the perspective of this function, it can support such a condition.